beaconChain/tests/helpers/stateTransitionHelpers.test.ts

```diff
@@ -542,3 +549,118 @@ describe("merkleRoot", () => {
     assert(expectedRoot.equals(computedRoot), `${expectedRoot} didn't equal ${computedRoot}`);
   });
 });
+
+
+describe("intToBytes", () => {
+  it("should convert number 0 to an empty buffer", () => {
+    const res = intToBytes(0, 1)
+    assert(res.equals(Uint8Array.from([0])), `got: ${res}, expected:${[0]}`)
+  });
+  it("should convert BigNumber 0 to an empty buffer", () => {
+    const res = intToBytes(new BN(0), 1)
+    assert(res.equals(Uint8Array.from([0])), `got: ${res}, expected:${[0]}`)
+  });
+  it("should convert number 1 to a single byte 0b00000001", () => {
+    const res = intToBytes(1, 1)
+    assert(res.equals(Uint8Array.from([1])), `got: ${res}, expected:${[1]}`)
+  });
+  it("should convert BigNumber 1 to a single byte 0b00000001", () => {
+    const res = intToBytes(new BN(1), 1)
+    assert(res.equals(Uint8Array.from([1])), `got: ${res}, expected:${[1]}`)
+  });
+  it("should convert number 16704 to two bytes [0x40, 0x42]", () => {
+    const res = intToBytes(16704, 2)
+    assert(res.equals(Uint8Array.from([0x40, 0x41])), `got: ${res}, expected:${[0x40, 0x41]}`)
+  });
+  it("should convert BigNumber 16704 to two bytes [0x40, 0x42]", () => {
+    const res = intToBytes(new BN(16704), 2)
+    assert(res.equals(Uint8Array.from([0x40, 0x41])), `got: ${res}, expected:${[0x40, 0x41]}`)
+  });
+});
+
+
+describe("getRandaoMix", () => {
+  it("should return first randao mix for GENESIS_EPOCH", () => {
+    // Empty state in 2nd epoch
+    const state = generateState({
+      slot: GENESIS_SLOT.addn(SLOTS_PER_EPOCH),
+      latestRandaoMixes: [Buffer.from([0xAB]), Buffer.from([0xCD])]
+    })
+    const res = getRandaoMix(state, GENESIS_EPOCH)
+    assert(res.equals(Uint8Array.from([0xAB])))
+  })
+  it("should return second randao mix for GENESIS_EPOCH + 1", () => {
+    // Empty state in 2nd epoch
+    const state = generateState({
+      slot: GENESIS_SLOT.addn(SLOTS_PER_EPOCH * 2),
+      latestRandaoMixes: [Buffer.from([0xAB]), Buffer.from([0xCD]), Buffer.from([0xEF])]
+    })
+    const res = getRandaoMix(state, GENESIS_EPOCH.addn(1))
+    assert(res.equals(Uint8Array.from([0xCD])))
+  })
+  it("should fail to get randao mix for epoch more than LATEST_RANDAO_MIXES_LENGTH in the past", () => {
+    // Empty state in epoch LATEST_RANDAO_MIXES_LENGTH with incrementing randao mixes
+    const state = generateState({
+      slot: GENESIS_SLOT.addn(SLOTS_PER_EPOCH * LATEST_RANDAO_MIXES_LENGTH),
+      latestRandaoMixes: Array.from({length: LATEST_RANDAO_MIXES_LENGTH}, (e, i) => Buffer.from([i]))
+    })
+    try {
+      getRandaoMix(state, GENESIS_EPOCH)
+      assert.fail('Expected error to be thrown')
```

Comment:
we can use `assert.throws(() => getRandaoMix(state, GENESIS_EPOCH))` if we expect it to throw an error